CONTRIBUTING.md

```diff
@@ -71,6 +71,32 @@ As a quick reference, the format of each page should match the following templat
 For more detailed page formatting guidelines,
 refer to the [style guide](contributing-guides/style-guide.md).
 
+## Translations
+
+Translation of pages can be done by simply creating the corresponding page in the appropriate top-level language specific directory under the appropriate platform subdirectory.
```

Comment:
We can make the sentence a bit easier to chew by forgoing some accuracy, since that's explained in more detail below. Here's what I'd suggest:

```suggestion
Translation of pages can be done by simply creating the corresponding page in the appropriate language-specific directory.
```